qiskit_nature/second_q/problems/electronic_properties_container.py

```diff
@@ -0,0 +1,72 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The electronic properties container."""
+
+from __future__ import annotations
+
+from typing import cast
+
+from qiskit_nature.second_q.properties import (
+    AngularMomentum,
+    ElectronicDipoleMoment,
+    Magnetization,
+    ParticleNumber,
+)
+
+from .properties_container import PropertiesContainer
+
+
+class ElectronicPropertiesContainer(PropertiesContainer):
+    """The container class for electronic structure properties."""
+
+    @property
+    def angular_momentum(self) -> AngularMomentum | None:
+        """Returns the :class:`qiskit_nature.second_q.properties.AngularMomentum` property."""
+        return cast(AngularMomentum, self._properties.get("AngularMomentum", None))
```

Comment:
In this `self._properties.get("AngularMomentum", None` since elsewhere the class name is used would this be better as AngularMomentum.__name__ to avoid hard coding the string. Maybe some simple _getter  could be done like the setter to centralize/simplify things a little perhaps?

Reply:
Yeh I should have though of checking the getters for consistency, my bad :see_no_evil: 
Will update this tomorrow, too :+1: